pkg/idp/validations/passwordValidator_test.go

```diff
@@ -0,0 +1,64 @@
+package validations
+
+import (
+	"testing"
+)
+func TestPasswordValidator(t *testing.T) {
```

Comment:
I like the use of the std testing pkg, but in OCM we use ginkgo and gomega.

Can you update these tests to follow similar please :pray: 

Example - https://github.com/openshift-online/ocm-common/blob/main/pkg/cluster/validations/cluster_node_test.go